### PR TITLE
Fix multiple tests for the Firefox browser

### PR DESCRIPTION
**Pull request by dvkruchinin**
### Motivation and context
A warning window appears about an unsupported platform when using cy.visit() or cy.reload()
Adding a css class to minimize the use of ant-* classes.

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2020 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by dvkruchinin** [comment]
@ActiveChooN Hi,
Please take a look this PR.

File: tests/cypress/integration/actions_users/issue_1810_login_logout.js [hunk 1]
```diff
@@ -64,7 +74,12 @@ context('When clicking on the Logout button, get the user session closed.', () =
                 responce = await responce['headers']['set-cookie'];
                 const csrfToken = responce[0].match(/csrftoken=\w+/)[0].replace('csrftoken=', '');
                 const sessionId = responce[1].match(/sessionid=\w+/)[0].replace('sessionid=', '');
-                cy.visit(`/login-with-token/${sessionId}/${csrfToken}?next=/tasks/${taskId}`)
+                cy.visit(`/login-with-token/${sessionId}/${csrfToken}?next=/tasks/${taskId}`);
+                if (Cypress.browser.family !== 'chromium') {
+                    cy.get('.cvat-modal-unsupported-platform-warning').within(() => {
+                        cy.contains('button', 'OK').click();
+                    });
+                }
```
Comment (bsekachev) [inline tests/cypress/integration/actions_users/issue_1810_login_logout.js:82]: Can we put the code to cypress command instead of duplication?

Comment (dvkruchinin) [reply tests/cypress/integration/actions_users/issue_1810_login_logout.js]: Sure. Reworked. Thanks.

File: tests/cypress/integration/actions_users/issue_1810_login_logout.js [hunk 2]
```diff
@@ -10,17 +10,19 @@ context('When clicking on the Logout button, get the user session closed.', () =
     const issueId = '1810';
     let taskId;
 
+    function closeModal() {
+        if (Cypress.browser.family !== 'chromium') {
```
Comment (bsekachev) [inline tests/cypress/integration/actions_users/issue_1810_login_logout.js:14]: Can we put the condition withing closeModalUnsupportedPlatform?

Comment (dvkruchinin) [reply tests/cypress/integration/actions_users/issue_1810_login_logout.js]: Yes. Done.

**Review by bsekachev — APPROVED** [review]

**Pull request merged**